src/lib/token_list_gen.ts

```diff
@@ -219,7 +220,9 @@ export const generateTokenList = async (
         // parse null terminated bytes32 strings
         else if (_name.length === 64)
           return utils.parseBytes32String('0x' + _name);
-        else return _name;
+        else if (token.l2Address === USDCContractAddressArb1) {
+          return 'Bridged USDC (Arb1)';
```

Comment:
Currently, name is `USD Coin (Arb1)`. Changing it to `USD Coin bridged (Arb1)` would be over the 20 characters limit we have.